### PR TITLE
Update qrest.1

### PR DESCRIPTION
**Pull request by trebmuh**
améliorations

### CODE REVIEW

File: doc/manpages/man1/qrest.1 [hunk 1]
```diff
@@ -11,7 +11,7 @@ There are 2 ways to entered the tempo value :
 .IP \(bu 2
 Keyboard numerical input : User simply types tempo value in a text field and hit \'enter\'.
 .IP \(bu 2
-Realtime button click : User clicks a button in sync with beats, and qrest will tell her if she taps steadily.
+Realtime button click : User clicks a button in sync with beats, and qrest will tell her/him if she/he taps steadily.
```
Comment (deufrai) [inline doc/manpages/man1/qrest.1]: using she/her is the usual 'polite' way of refering to anyone, male or female. I won't merge this change

**Review by deufrai — CHANGES_REQUESTED** [review]

File: doc/manpages/man1/qrest.1 (hunk 1 above)
Comment (trebmuh) [reply doc/manpages/man1/qrest.1]: Ah ? never saw that before. any link?. Anyway, I don't care about that and I just remove that change.

Comment (deufrai) [reply doc/manpages/man1/qrest.1]: See : http://english.stackexchange.com/questions/28508/reason-for-the-current-trend-to-use-she-as-the-gender-neutral-pronoun

**Review by deufrai — APPROVED** [review]

File: doc/manpages/man1/qrest.1 (hunk 1 above)
Comment (trebmuh) [reply doc/manpages/man1/qrest.1]: Ah ok, a feminism VS machismo stuff. So, I really don't care :)
I took it as a mistake first, but if it's done on purpose. Then, I reversed it. And this PR is all good to be merged in.

**Pull request closed without merging**